wireapp/wire-android-reloaded
# feat: add asset message placeholder [AR-1150]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1150" title="AR-1150" target="_blank"><img alt="Subtask" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10816?size=medium" />AR-1150</a>  Implement preview behavior for displaying images vs files on conversation +
  </td></table>
  <br />
 
















----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Previously, we could only send normal text and image messages. Now we can also attach and send any other type of file, which will be treated as generic file assets. 

For now we can only display the generic asset place holder, the "Tap to download" behavior will be added later on a PR following this one.

### Dependencies (Optional)

This will need to be merged once [the kalium counterpart PR](https://github.com/wireapp/kalium/pull/404) has been merged as well.

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

Just open any given conversation, choose "attach file" or "record video" and once picked, check that the file has been sent successfully.

### Attachments (Optional)

https://user-images.githubusercontent.com/2468164/163402196-5200a285-7bc1-4b50-ab16-9cb746609eee.mov

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
